src/app/core/interceptors/error-notifications/error-notifications.service.ts

```diff
@@ -34,20 +34,32 @@ export class ErrorNotificationsInterceptor implements HttpInterceptor {
             return;
           }
 
-          if (!!errorInstance.error && !!errorInstance.error.error) {
-            this._notificationService.error(
-              `Error ${errorInstance.status}: ${
-                errorInstance.error.error.message || errorInstance.message || errorInstance.statusText
-              }`
-            );
-          } else if (
-            errorInstance.message &&
-            this._silenceErrArr.every(partial => !errorInstance.message.includes(partial))
-          ) {
-            this._notificationService.error(`${errorInstance.message}`);
+          if (this._shouldSilenceError(errorInstance)) {
```

Comment:
Can we silence the error only if the feature is disabled? Otherwise, we might silence something that should be visible.

Reply:
I'd rather make partial messages more detailed. Everything is handled now in a generic way and it would be hard to keep it that way if we have to check for an exact single error and additionally asynchronously get admin settings to find out if some feature is enabled/disabled.

Reply:
Ok, sure.